recipes/ogdf/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ogdf" %}
+{% set version = "2020.02" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ogdf/ogdf/archive/catalpa-202002.tar.gz
+  sha256: 32f3f943675e55d1a23e62293d016dc1644b1176271d60f41788ade60c2d3fc5
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_subpackage('ogdf', max_pin='2020.02') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.1
+    - make  # [not win]
+    - ninja  # [win]
+
```

Comment:
This recipe needs some tests.  Even basic existence tests: https://conda-forge.org/docs/maintainer/adding_pkgs.html#simple-existence-tests